examples/tutorials/get-started/quickstart.ipynb

```diff
@@ -102,9 +118,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 2. Create a new run\n",
+    "## 2. Test the flow\n",
     "\n",
-    "`web-classification` is a flow demonstrating multi-class classification with LLM. Given an url, it will classify the url into one web category with just a few shots, simple summarization and classification prompts."
+    "With a given flow, you can easily test it with `pf.test()` function. It helps to test the basic functionality of your flow and ensure that it's working as expected. Besides using existing flow, you can also create a new flow. More details are in [initialize and test a flow](../../../docs/how-to-guides/init-and-test-a-flow.md).\n",
+    "\n",
+    "### Flow directory\n",
+    "\n",
+    "Every flow exists as a flow directory in local. This flow directory contains all contents of a flow. The entry file is `flow.dag.yaml` which describes the DAG(Directed Acyclic Graph) of a flow. Note there is default value for flow inputs so you can use these default values to test the flow.:\n",
+    "\n",
```

Comment:
we can reference only 1 screenshot image here. 
2 images would be too much for samples

Reply:
Maybe we just add links back to doc site and keep sample simple.